pkg/api/customization/podsecuritypolicytemplate/validator.go

```diff
@@ -0,0 +1,32 @@
+package podsecuritypolicytemplate
+
+import (
+	"errors"
+	"strings"
+
+	"github.com/rancher/norman/httperror"
+	"github.com/rancher/norman/types"
+	"github.com/rancher/norman/types/convert"
+	"k8s.io/apimachinery/pkg/util/validation/field"
+	"k8s.io/kubernetes/pkg/apis/policy"
+	"k8s.io/kubernetes/pkg/apis/policy/validation"
+)
+
+// Validator uses k8s Pod Security Policy validation to prevent creation of pod security policy templates that are invalid
+func Validator(request *types.APIContext, schema *types.Schema, data map[string]interface{}) error {
+	var spec policy.PodSecurityPolicySpec // k8s psp
+	if err := convert.ToObj(data, &spec); err != nil {
+		return httperror.WrapAPIError(err, httperror.InvalidBodyContent, "Pod Security Policy spec conversion error")
+	}
+	var allErrs field.ErrorList
+	allErrs = validation.ValidatePodSecurityPolicySpec(&spec, &field.Path{})
+	if len(allErrs) > 0 { // concatenate all errors to present in UI
+		strs := make([]string, len(allErrs))
+		for i, v := range allErrs {
+			strs[i] = v.Detail
+		}
+		return httperror.WrapAPIError(errors.New(allErrs[0].Type.String()), httperror.InvalidBodyContent, strings.Join(strs, ", "))
```

Comment:
how is this going to look in the UI? I feel like it is going to be an unreadable run-on. Could you possibly take a screenshot and attach to this PR?

Reply:
<img width="1489" alt="Concatenated with ," src="https://user-images.githubusercontent.com/31839142/55650859-bc2b1c00-579b-11e9-8baa-956b478f8972.png">
@vincent99 Took a look at this when Nathan and I were working on it. He LGTM'edthe UI appearance.

Reply:
We don't have a standard mechanism for returning multiple errors so concat was the default fallback..  I don't think having many different errors at once is going to be common enough to worry much about how readable it is

..Though if you stick a newline between each one in the string, we could make them display as multiple lines...